src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,27 @@
 const generalCSQuiz = [
+  {
+    Question:
+      "Which sorting algorithm has an average and worst-case time complexity of O(nlogn)?",
```

Comment:
worst case would be O(n^2) depending on the selection of the pivot element
so the question should be updated to reflect that

```suggestion
      "Which sorting algorithm has an average time complexity of O(nlogn)?",
```